packages/editor/src/components/editor/Editor.tsx

```diff
@@ -68,14 +67,6 @@ export const Editor = (props: FormContext) => {
     <AppProvider value={{ data: data.data, setData: mutation.mutate, selectedElement, setSelectedElement, ui, setUi }}>
       <DndContext>
         <ResizablePanelGroup direction='horizontal' autoSaveId='form-editor-resize'>
-          {ui.components && (
```

Comment:
did you also remove this component state?

Reply:
No, I forgot, but I have removed it now